sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/entries/CreateNewTravelEntryPage.java

```diff
@@ -39,4 +39,11 @@
   public static final By POINT_OF_ENTRY_DETAILS_INPUT =
       By.cssSelector(".v-window #pointOfEntryDetails");
   public static final By SAVE_BUTTON = By.cssSelector(".v-window #commit");
+  public static final By PICK_OR_CREATE_PERSON_TITLE_DE =
+      By.xpath("//div[@class='v-window-header']");
+  public static final By SAVE_POPUP_CONTENT = By.cssSelector(".popupContent #commit");
+  public static final By PICK_A_EXISTING_PERSON_LABEL_DE =
+      By.xpath("//*[text()='W\u00E4hlen Sie eine passende Person']");
+  public static final By PICK_A_EXISTING_CASE_LABEL_DE =
```

Comment:
isn't the EN version working as well for this? to have a common ground for them

Reply:
That kind of string we don't have in EN version: W\u00E4hlen Sie eine passende Person